### ISSUE TITLE
Interpretation question / nitpick in ch1

### ISSUE THREAD

**Issue by keflavich** [Open-Astrophysics-Bookshelf/star_formation_notes#14]
https://github.com/Open-Astrophysics-Bookshelf/star_formation_notes/blob/fc6dac3a31fecec319fe129a72eeb73b1b483de6/chapters/chapter1.tex#L36

> The gas, in contrast, does not emit thermal radiation because it is nowhere near dense enough to reach equilibrium with the radiation field.

No matter how dense the gas is, it will not emit thermal radiation, no?  Neutral atomic and molecular gas is always constrained to emit in narrow lines.  Might it be better to rephrase this as:

> The gas, in contrast, does not emit thermal radiation because neutral atoms and molecules are only able to emit through narrow bands in the energy spectrum.

I hope I'm not missing something obvious here.  It may be a language issue, that I'm interpreting 'thermal' here as 'continuous, quasi-blackbody' while the intended meaning is 'with all energy levels populated', but I think this sentence could nevertheless be rephrased for clarity.

(I'm re-reading sections as I prepare to teach to undergraduates; this is non-urgent)

**Comment by markkrumholz** [comment]
I guess this is a bit of a pedantic issue. Part of it is what is meant by thermal; we talk all the time about lines being thermalized, in the sense that the density is high enough that at line center T_kinetic = T_brightness, and the observed intensity is equal to that of a blackbody. So the emission can be thermal in that sense.

In fact, it can even become thermal in the continuum sense for sufficiently large volume or column density, because line broadening mechanisms can cause the lines to overlap and form a pseudo-continuum. This happens in practice for example with H2 molecules in the Lyman-Werner bands, where at high column the line wings of the various transitions overlap, and you more or less get a continuum opacity. It happens in stellar winds in the UV too, where you get line blanketing from a forest of iron lines that are all Doppler-broadened as the wind accelerates, and so the effective opacity as viewed from the outside looks like a continuum. Pressure broadening in the atmospheres of stars can also create pseudo-continuua in some parts of the spectrum. So I wouldn't agree with the categorical statement that gas is always constrained to emit in narrow lines; that is only true when the lines aren't broadened to the point where they begin overlapping.

That said, I agree that the current phrasing might be somewhat confusing. How about "By contrast, under interstellar conditions the gas can only emit and absorb radiation in narrow lines". This avoid ambiguities in the definition of thermalization, which gets tricky to define in a really general sense.

**Comment by keflavich** [comment]
Yes, I agree with your rephrasing.  The important thing to contrast is broad-band continuous radiation from narrow-line emission; the latter is still thermal in many cases.
